.github/workflows/docusaurus.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy Docusaurus
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+    - 'stream-chat-swift-docs/**' # Run when any doc file is changed
+    - 'DocsSnippets/**' # Also re-run when we update doc snippets
+
+jobs:
+  deploy-docusaurus:
+    name: 'Deploy Docusaurus'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install Bot SSH Key
+        uses: webfactory/ssh-agent@v0.4.1
+        with:
+          ssh-private-key: ${{ secrets.BOT_SSH_PRIVATE_KEY }}
+      - name: Install yarn (Somehow it isn't pre-installed?)
```

Comment:
IMO it's better to channel our frustrations in our Slack channel rather than here ;-) Or at least make it a comment and don't include it into the action name?

Reply:
totally, apologizes 😅 most probably it _is_ preinstalled and we'll remove this action, somehow it didn't work locally